pkg/api/pod/util.go

```diff
@@ -328,6 +329,13 @@ func DropDisabledPodFields(pod, oldPod *api.Pod) {
 	dropPodStatusDisabledFields(podStatus, oldPodStatus)
 }
 
+// AddDisabledFields to the provided set of resetFields
```

Comment:
Does `to` mean `resetFields`? I recommend making this documentation a little clearer or changing the parameter name, even though it's very clever right now... maybe TOO clever :)